cineast-runtime/src/main/resources/log4j2.xml

```diff
@@ -5,8 +5,9 @@
   </Properties>
   <Appenders>
     <Console name="Console" target="SYSTEM_OUT">
-      <PatternLayout pattern="%d{yyyy-MM-dd HH:mm:ss.SSS} [%t] %p %c{1.} - %msg%n"/>
-      <ThresholdFilter level="DEBUG" onMatch="ACCEPT" onMismatch="DENY"/>
+      <PatternLayout disableAnsi="false"
+        pattern="%highlight{%d{yyyy-MM-dd HH:mm:ss.SSS} [%t] %p %c{1.} - %-5level:%msg%n}{FATAL=Red, ERROR=Red, WARN=Yellow, INFO=Green, DEBUG=Blue, TRACE=White}"/>
+      <ThresholdFilter level="TRACE" onMatch="ACCEPT" onMismatch="DENY"/>
```

Comment:
I would prefer if the default log level was changed back to `DEBUG`, `TRACE` outputs too much information that isn't useful in most use cases.